extension/test/e2e/pageObjects/PaypalPopupPage.js

```diff
@@ -14,7 +14,10 @@ export default class PaypalPopupPage {
     await this.page.type('#password', '!nh-NNS1')
     await this.page.click('#btnLogin')
 
+    await this.page.waitForTimeout(5000)
     await this.page.waitForSelector('#payment-submit-btn')
```

Comment:
Does it make sense to wait for selector below if you wait already 5 sec? If yes, then why?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1049/commits/4dad4d732ae2904866d3875141e5bc24bc555244